llvm/include/llvm/Demangle/ItaniumDemangle.h

```diff
@@ -2793,7 +2793,7 @@ template <typename Derived, typename Alloc> struct AbstractManglingParser {
   Node *parseClassEnumType();
   Node *parseQualifiedType();
 
-  Node *parseEncoding();
+  Node *parseEncoding(bool ParseParams = true);
```

Comment:
This file has a big comment at the top about it being a duplicate of code elsewhere in the LLVM tree. You'll need to action the comment, so that the two versions don't get out-of-sync.

Reply:
> There are two copies of this file in the source tree.  The one under libcxxabi is the original and the one under llvm is the copy.  Use cp-to-llvm.sh to update the copy.

Updated llvm-project/libcxxabi/src/demangle/ItaniumDemangle.h too.